instances/technique/biocytinStaining.jsonld

```diff
@@ -4,11 +4,14 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/biocytinStaining",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "In 'biocytin staining' the chemical compound biocytin is used to highlight morphological details of nerve cells.",
+  "description":  "Biocytin staining is a technique commonly used in combination with intracellular electrophysiology for post-hoc recovery of morphological details of the studied neurons. For this, the chemical compound biocytin is included in the electrode in order to fill the studied cell. It allows for the visualisation of the dendritic arborization and the regions targeted by the axons of the studied neurons.",
```

Comment:
I've changed "recorded" to "studied". As far as I know also stimulated neurons can be filled.